packages/team-management/src/components/TeamsAddEdit/Form.tsx

```diff
@@ -0,0 +1,196 @@
+import React, { useState } from 'react';
+import { Select, Button, Form as AntdForm, Radio, Input, notification } from 'antd';
+import { history } from '@onaio/connected-reducer-registry';
+import { MinusCircleOutlined, PlusOutlined } from '@ant-design/icons';
+import { v4 } from 'uuid';
+import { API_BASE_URL, TEAMS_POST, PRACTITIONER_POST } from '../../constants';
+import { OpenSRPService } from '@opensrp/server-service';
+import { OrganizationPOST } from '../../ducks/organizations';
+import { Practitioner, PractitionerPOST } from '../../ducks/practitioners';
+
+const layout = { labelCol: { span: 8 }, wrapperCol: { span: 11 } };
+const offsetLayout = { wrapperCol: { offset: 8, span: 11 } };
+const layoutFull = { labelCol: { span: 8 }, wrapperCol: { span: 16 } };
+const offsetLayoutFull = { wrapperCol: { offset: 8, span: 16 } };
+
+export interface FormField {
+  name: string;
+  active: boolean;
+  practitioners: string[];
+}
+
+interface Props {
+  id?: string;
+  practitioner: Practitioner[];
+  initialValue?: FormField | null;
+  accessToken: string;
+}
+
+/**
+ * Handle form submission
+ *
+ * @param {string} accessToken Token for api calles
+ * @param {Object} values the form fields
+ * @param {Practitioner} practitioner list of practitioner to filter the selected one from
+ * @param {Function} setIsSubmitting function to set IsSubmitting loading process
+ */
+export async function onSubmit(
```

Comment:
@ashfahan seems you forgot to add the request for editing teams.

Reply:
Resolved